runtime/bcverify/staticverify.c

```diff
@@ -608,6 +608,12 @@ checkBytecodeStructure (J9CfrClassFile * classfile, UDATA methodIndex, UDATA len
 		case CFR_BC_jsr:
 			method->j9Flags |= CFR_J9FLAG_HAS_JSR;
 			classfile->j9Flags |= CFR_J9FLAG_HAS_JSR;
+
+			PEEK_U16(index, bcIndex);
```

Comment:
Rather than this change, it might be much simpler to just have a boolean here which disallows 0 in the fallthrough cases. No new macros, and less than 10 lines of new code. As only the jsr bytecodes would set it to true, you can still use the new NLS message in the 0 path.